src/umpire/strategy/AllocationTracker.cpp

```diff
@@ -61,7 +61,14 @@ AllocationTracker::getHighWatermark() const noexcept
 std::size_t
 AllocationTracker::getActualSize() const noexcept
 {
-  return m_allocator->getActualSize();
+  return (m_allocator->getActualSize() > 0) ? 
+    m_allocator->getActualSize() : m_current_size;
+}
```

Comment:
Not sure if the compiler will wind of up calling method twice or not.

```suggestion
auto actual_size = m_allocator->getActualSize();
return actual_size > 0 ? actual_size : m_current_size;
```